contextid_test.go

```diff
@@ -22,8 +22,9 @@ func getIdFromRequest(t *testing.T, server *httptest.Server) uint {
 
 func TestContextId(t *testing.T) {
 	handler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		id := r.Context().Value("id")
-		_, err := w.Write([]byte(strconv.FormatUint(uint64(id.(uint)), 10)))
+		id, ok := (r.Context().Value("id")).(uint)
```

Comment:
nit: Redundant parentheses.

Reply:
Done.

Fixed in 6f754210a8dcc37c5f164656f75c22a5a6c8449d